### PR TITLE
@86eqxnrqc #Fix customer creation minor bugs

### PR DESCRIPTION
**Pull request by BryanPalad**

### CODE REVIEW

**Review by acostales08 — APPROVED** [review]

File: packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts [hunk 1]
```diff
@@ -24,7 +24,10 @@ export const registrationSchema = yup.object({
   lastname: yup.string().required("Last Name is required").default(""),
   email: yup
     .string()
-    .email("Please provide a valid email")
+    .matches(
+      /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
```
Comment (JMSevilla) [inline packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts:28]: You can place this regex to `fp-regex.ts` but can you also please change the file name from `fp-regex.ts` to `regex.ts` 

note : once changed please check for any implementation that relates to `fp-regex.ts`

File: packages/core-library/components/forms/TextField.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ interface Props<T extends object> {
   disabled?: boolean;
   multiline?: boolean;
   rows?: number;
-  IsRegister?: boolean;
+  isRegister?: boolean;
```
Comment (JMSevilla) [inline packages/core-library/components/forms/TextField.tsx:49]: This property is being logged as error from the next application as it has a uppercase on it. can you change this to `isregister`

**Review by JMSevilla — CHANGES_REQUESTED** [review]

File: packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts [hunk 3]
```diff
@@ -24,7 +25,7 @@ export const registrationSchema = yup.object({
   lastname: yup.string().required("Last Name is required").default(""),
   email: yup
     .string()
-    .email("Please provide a valid email")
+    .matches(EMAIL_REGEX, "Please provide a valid email")
```
Comment (JMSevilla) [inline packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts:28]: Sorry I overlook this one, we have a built-in function from yup we can use instead to use regex

kindly use .email()

File: packages/core-library/utils/regex.ts [hunk 4]
```diff
@@ -2,3 +2,4 @@ export const OTP_REGEX = /^\d{6}$/;
 export const CONTAINS_NUMBER_REGEX = /(?=.*[0-9])/;
 export const CONTAINS_UPPERCASE_REGEX = /(?=.*[A-Z])/;
 export const STRING_REGEX = /^[ A-Za-z0-9_'*(),./#&+-]*$/;
+export const EMAIL_REGEX = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```
Comment (JMSevilla) [inline packages/core-library/utils/regex.ts:5]: Remove this since it is not necessary

**Review by JMSevilla — CHANGES_REQUESTED** [review]

File: packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts (hunk 3 above)
Comment (BryanPalad) [reply packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts:28]: The .email() function from Yup was previously implemented, but it still accepts emails like example@gmail without a .com or valid domain, making the validation inconsistent and allowing registration to proceed without a valid email. That's why I've used custom email regex.

Comment (JMSevilla) [reply packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts:28]: In that case can you combine both .email() & regex

Comment (BryanPalad) [reply packages/core-library/system/app/internal/blocks/Account/Registration/validation.ts:28]: Copy that

**Review by JMSevilla — APPROVED** [review]

**Review by JezChristian — APPROVED** [review]

**Pull request merged**